app/tests/components_tests/test_forms.py

```diff
@@ -190,3 +196,24 @@ def test_image_widget_current_value_in_archive_item_and_display_set_update_forms
         .pk
         == user_upload.pk
     )
+
+
+@pytest.mark.parametrize(
+    "component_interface_kind",
+    [kind for kind in INTERFACE_TYPE_JSON_EXAMPLES.keys()],
+)
+@pytest.mark.parametrize("store_in_db", [True, False])
+@pytest.mark.django_db
+def test_interface_form_field_help_text_example_download_link(
+    component_interface_kind, store_in_db
+):
+    user = UserFactory()
+    ci = ComponentInterfaceFactory(
+        kind=component_interface_kind, store_in_database=store_in_db
+    )
+    field = InterfaceFormField(instance=ci, user=user, form_data={})
+
+    encoded_example = quote(json.dumps(ci.json_kind_example.value, indent=2))

```

Comment:
I would make this explicit. Add a few concretely encoded examples and provide them via the parametrize. This seems to be too 'smart'.

I think it is OK if does not test every single INTERFACE_TYPE_JSON_EXAMPLES, I would focus the test on three cases:
- Should show an example
- Should not show an example
- Special case of the first, but not a JSONField: so no example

Reply:
Done